debian/enterprise-kafka/Dockerfile

```diff
@@ -36,9 +36,10 @@ RUN echo "===> installing confluent-support-metrics ..." \
     && echo "===> clean up ..."  \
     && apt-get clean && rm -rf /tmp/* /var/lib/apt/lists/* \
     \
-		&& echo "===> Setting up ${COMPONENT} dirs..." \
-   	&& mkdir -p /var/lib/${COMPONENT}/data /etc/${COMPONENT}/secrets\
-   	&& chmod -R ag+w /etc/${COMPONENT} /var/lib/${COMPONENT}/data /etc/${COMPONENT}/secrets
+    && echo "===> Setting up ${COMPONENT} dirs..." \
+    && mkdir -p /var/lib/${COMPONENT}/data /etc/${COMPONENT}/secrets\
+    && chmod -R ag+w /etc/${COMPONENT} /var/lib/${COMPONENT}/data /etc/${COMPONENT}/secrets \
+    && chown root:root /var/log/kafka /var/log/confluent /var/lib/kafka /var/lib/zookeeper
```

Comment:
should probably be recursive: `chown -R ..`

Reply:
Note that your original change doesn't use recursive, so they are only changed for the top level and not necessary here.
There are subdirs still owned by root, but they are empty and also world writeable. 

I will update this to use recursive, but you might want to do the same in kafka-packaging if it affects systemd/security.